### PR TITLE
Initial wiring for websocket connection

### PR DESCRIPTION
**Pull request by anishj0shi**

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -31,3 +31,7 @@ required = [
 [[override]]
   name = "k8s.io/code-generator"
   version = "kubernetes-1.12.9"
+
```
Comment (antoineco) [inline Gopkg.toml]: Let’s move it at the beginning to have direct dependencies defined before transitive dependencies. It improves clarity.

File: adapter/adapter.go [hunk 2]
```diff
@@ -0,0 +1,16 @@
+package adapter
```
Comment (antoineco) [inline adapter/adapter.go]: Don’t forget copyright headers, the Apache license requires it.

File: cmd/mqttsource-service/main.go [hunk 3]
```diff
@@ -0,0 +1,69 @@
+package main
```
Comment (antoineco) [inline cmd/mqttsource-service/main.go]: Would you mind renaming the cmd to “mqtttsource-adapter”? It’s the terminology used by Knative sources.

Comment (anishj0shi) [reply cmd/mqttsource-service/main.go]: not for the service. for service they use package main

File: adapter/adapter.go (hunk 2 above)
Comment (anishj0shi) [reply adapter/adapter.go]: how do we do that?

Comment (antoineco) [reply adapter/adapter.go]: Just copy from any other file in the project.

File: Gopkg.toml (hunk 1 above)
Comment (anishj0shi) [reply Gopkg.toml]: done

File: adapter/adapter.go (hunk 2 above)
Comment (anishj0shi) [reply adapter/adapter.go]: done

File: cmd/mqttsource-service/main.go (hunk 3 above)
Comment (anishj0shi) [reply cmd/mqttsource-service/main.go]: done

**Pull request merged**